io_scene_niftools/modules/nif_import/animation/__init__.py

```diff
@@ -54,6 +55,23 @@ def __init__(self):
         # and still be able to access existing actions from this run
         self.actions = {}
 
+    @staticmethod
+    def get_controller_data(ctrl):
+        """Return data for ctrl, look in interpolator (for newer games) or directly on ctrl"""
+        if ctrl.interpolator:
+            data = ctrl.interpolator.data
+        else:
+            data = ctrl.data
+        # these have their data set as a KeyGroup on data
+        if isinstance(data, (NifFormat.NiBoolData, NifFormat.NiFloatData, NifFormat.NiPosData)):
```

Comment:
Do we need to care about data on the wrapping objects.

Reply:
No. Basically, this just gets the keyframe data. Anything of relevance is generally on the controller. The interpolators etc are not interesting.

Reply:
How boring 😞